examples/pxScene2d/external/node-v8.15.1_mods.patch

```diff
@@ -1,324 +1,5 @@
-diff --git a/examples/pxScene2d/external/libnode-v8.15.1/lib/internal/bootstrap_node.js b/examples/pxScene2d/external/libnode-v8.15.1/lib/internal/bootstrap_node.js
-index 0e9c36c..66e98d3 100644
---- a/examples/pxScene2d/external/libnode-v8.15.1/lib/internal/bootstrap_node.js
-+++ b/examples/pxScene2d/external/libnode-v8.15.1/lib/internal/bootstrap_node.js
-@@ -211,7 +211,9 @@
-         perf.markMilestone(
-           NODE_PERFORMANCE_MILESTONE_PRELOAD_MODULE_LOAD_END);
-         // If -i or --interactive were passed, or stdin is a TTY.
--        if (process._forceRepl || NativeModule.require('tty').isatty(0)) {
-+/* MODIFIED CODE BEGIN */
-+//      if (process._forceRepl || NativeModule.require('tty').isatty(0)) {
-+/* MODIFIED CODE END */
-           // REPL
-           const cliRepl = NativeModule.require('internal/repl');
-           cliRepl.createInternalRepl(process.env, function(err, repl) {
-@@ -233,6 +235,8 @@
-             // User passed '-e' or '--eval'
-             evalScript('[eval]');
-           }
-+/* MODIFIED CODE BEGIN */
-+/*
-         } else {
-           // Read all of stdin - execute it.
-           process.stdin.setEncoding('utf8');
-@@ -250,7 +254,8 @@
-               evalScript('[stdin]');
-             }
-           });
--        }
-+        } */
-+/* MODIFIED CODE END */
-       }
-     }
-     perf.markMilestone(NODE_PERFORMANCE_MILESTONE_BOOTSTRAP_COMPLETE);
-diff --git a/examples/pxScene2d/external/libnode-v8.15.1/lib/net.js b/examples/pxScene2d/external/libnode-v8.15.1/lib/net.js
-index 98bcbc3..70b6414 100644
---- a/examples/pxScene2d/external/libnode-v8.15.1/lib/net.js
-+++ b/examples/pxScene2d/external/libnode-v8.15.1/lib/net.js
-@@ -34,6 +34,9 @@ const {
- const assert = require('assert');
- const cares = process.binding('cares_wrap');
- const uv = process.binding('uv');
-+/*MODIFIED CODE BEGIN*/
-+const fileSystem = require('fs');
-+/*MODIFIED CODE END*/
- 
- const { Buffer } = require('buffer');
- const TTYWrap = process.binding('tty_wrap');
-@@ -1083,6 +1086,27 @@ function lookupAndConnect(self, options) {
-     dnsopts.hints = dns.ADDRCONFIG;
-   }
- 
-+/*MODIFIED CODE BEGIN*/
-+  var ipMode = 0;
-+  try {
-+    fileSystem.accessSync('/tmp/ipmode_v4');
-+    ipMode = 4;
-+  } catch(e) {
-+    debug('/tmp/ipmode_v4 does not exist');
-+  }
-+
-+  try {
-+    fileSystem.accessSync('/tmp/ipmode_v6');
-+    ipMode = 6;
-+  } catch(e) {
-+    debug('/tmp/ipmode_v6 does not exist');
-+  }
-+
-+  if (ipMode == 4 || ipMode == 6) {
-+    dnsopts.family = ipMode;
-+  }
-+/*MODIFIED CODE END*/
-+
-   debug('connect: find host', host);
-   debug('connect: dns options', dnsopts);
-   self._host = host;
 diff --git a/examples/pxScene2d/external/libnode-v8.15.1/src/node.cc b/examples/pxScene2d/external/libnode-v8.15.1/src/node.cc
-index 3d791f0..d572a6d 100644
+index 3d791f0..0d810a9 100644
 --- a/examples/pxScene2d/external/libnode-v8.15.1/src/node.cc
 +++ b/examples/pxScene2d/external/libnode-v8.15.1/src/node.cc
```

Comment:
You should not modify files in libnode-v8.15.1. Add changes to node-v8.15.1_mods.patch.

Reply:
fine